ThunderCloud/MultiVideoPlayerViewController.swift

```diff
@@ -127,23 +127,32 @@ open class MultiVideoPlayerViewController: UIViewController {
 		
 		activityIndicator.startAnimating()
 	}
+    
+    private func getBottomSafeAreaInset() -> CGFloat {
```

Comment:
Don't really need to worry about catering for sub-iOS 11 any more! Has the Deployment Target not been bumped yet? If not, then leave as-is :)

Reply:
The target is still iOS 10 for this framework, I'll leave it as is - but then when it is bumped we'll go through and remove any  < iOS 11 handling code :)